README.md

```diff
@@ -59,7 +75,7 @@ You can run the local UI through [joshsim](https://language.joshsim.org/download
 $ java -jar joshsim.jar server
 ```
 
-This will start a local web server which makes the UI available via your browser where you can work in private. The local server supports a sandbox mode that limits access to only the code and jshd files provided, with no network access otherwise.
+This will start a local web server which makes the UI available via your browser where you can work in private. The local server supports a sandbox mode that limits access to only the code, jshd, and jshc files provided, with no network access otherwise.
```

Comment:
> The local server supports a sandbox mode that limits access to only the code, jshd, and jshc files provided, with no network access otherwise.

Let's do the following:

```
The local server supports a sandbox mode that limits access to only the code, jshd, and jshc files provided. However, the sandbox disallows network access otherwise.
```